CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 
+## StreamChat
+### 🐞 Fixed
+- Rare crash in `WebSocketPingController` [#3451](https://github.com/GetStream/stream-chat-swift/pull/3451)
```

Comment:
It is better that we are more specific, since there might be other crashes in the future about the `WebSocketPingController`
```suggestion
- Fix rare crash in `WebSocketPingController.connectionStateDidChange` [#3451](https://github.com/GetStream/stream-chat-swift/pull/3451)
```